src/SwiftRuntimeLibrary/src/ISwiftObject.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Swift.Runtime
+{
+    public unsafe interface ISwiftObject : IDisposable
```

Comment:
I suggest getting into the [habit of documenting public APIs](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/) during development. It helps with reviews and eventual public documentation efforts.

Reply:
If you set `GenerateDocumentationFile=true` in the csproj, you'll get warnings if you have undocumented APIs. Then setting `TreatWarningsAsErrors=true` and you'll never forget to document anyhing.

Reply:
Updated, thanks.